packages/baggage-span-processor/README.md

```diff
@@ -0,0 +1,42 @@
+# OpenTelemetry Baggage Span Processor for Node.js
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-url]
+
+The BaggageSpanProcessor reads entries stored in Baggage from the parent context and adds the baggage entries' keys and
```

Comment:
Adding a link to Baggage documentation would help users understand better what this is about
https://opentelemetry.io/docs/concepts/signals/baggage/